src/lib/components/chatInput/ChatMessageInput.svelte

```
@@ -41,6 +40,9 @@
   import { gotoWithSeachParams } from "$lib/utils/navigate";
   import { untrack } from "svelte";
   import { PremiumWrapper } from "../ui/premium-badge";
+  import { openCaptchaDialog, waitForTurnstile } from "$lib/client/services/turnstile.svelte";
```

ESLint

[error] 'waitForTurnstile' is defined but never used.

(@typescript-eslint/no-unused-vars)